.github/ISSUE_TEMPLATE/release.yml

```diff
@@ -40,8 +40,8 @@ body:
         - [ ] Increase version (versionPatch + 2) in `./vector/build.gradle`
         - [ ] Change the value of SDK_VERSION in the file `./matrix-sdk-android/build.gradle`
         - [ ] Commit and push `develop`
-        - [ ] Wait for [Buildkite](https://buildkite.com/matrix-dot-org/element-android/builds?branch=main) to build the `main` branch.
-        - [ ] Run the script `~/scripts/releaseElement.sh`. It will download the APKs from Buildkite check them and sign them.
+        - [ ] Wait for the [GitHub action](https://github.com/vector-im/element-android/actions/workflows/build.yml?query=branch%3Amain) to build the `main` branch.
+        - [ ] (TODO This script has to be updated) *Run the script `~/scripts/releaseElement.sh`. It will download the APKs from Buildkite check them and sign them.*
```

Comment:
Out of curiosity, where is this `releaseElement.sh` script?

Reply:
@mnaturel https://github.com/vector-im/element-android/pull/7583/commits/f75214648eeb853dcd3df226a9961c8498be0452

Can you check the script on your side?